v3-certificate/README.md

```diff
@@ -0,0 +1,15 @@
+# X.509 V3 Certificate Generator
+
+This sample script automatically generates an X.509 V3 Certificate using RSA and uploads it to the [Key Master](https://fusionauth.io/docs/v1/tech/core-concepts/key-master) in your FusionAuth instance.
```

Comment:
I'd add a note about why you might use this script. The reason is that if you use FusionAuth's generate functionality, it will only be a v2 certificate, right?